### PR TITLE
Add Sleep measurement

### PR DESCRIPTION
**Pull request by mborsz**
/assign @mm4tt

### CODE REVIEW

File: clusterloader2/pkg/measurement/common/sleep.go [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+}
+
+func createSleepMeasurement() measurement.Measurement {
+	return &sleepMeasurement{}
+}
+
+type sleepMeasurement struct{}
+
+func (s *sleepMeasurement) Execute(config *measurement.MeasurementConfig) ([]measurement.Summary, error) {
+	action, err := util.GetString(config.Params, "action")
```
Comment (wojtek-t) [inline clusterloader2/pkg/measurement/common/sleep.go]: Do we even need to introduce "action" parameter for this measurement?

Comment (mm4tt) [reply clusterloader2/pkg/measurement/common/sleep.go]: +1, let's keep it simple

**Comment by mm4tt** [comment]
Thanks, @mborsz !

**Comment by mm4tt** [comment]
Also travis seems to be failing with:
```
Boilerplate header is wrong for: /home/travis/gopath/src/k8s.io/perf-tests/clusterloader2/pkg/measurement/common/sleep.go
The command "./verify/verify-boilerplate.sh" exited with 1.
```

**Pull request merged**